src/StochasticDiffEq.jl

```diff
@@ -90,6 +91,8 @@ end
     seed_multiplier() = 1
   end
 
+  EnumX.@enumx AlgorithmInterpretation Ito Stratonovich
```

Comment:
This should be defined in SciMLBase.

Reply:
https://github.com/SciML/SciMLBase.jl/pull/777